pkg/pool-gyro/hardhat.config.ts

```diff
@@ -13,6 +13,11 @@ import 'hardhat-contract-sizer';
 import { warnings } from '@balancer-labs/v3-common/hardhat-base-config';
 
 const config: HardhatUserConfig = {
+  networks: {
```

Comment:
They're not deployable now then?

Reply:
it's to deploy the vault mock. The factories are within the limit.